diagnostic.md

```diff
@@ -8,6 +8,11 @@ Assume all code executes in strict mode.
 ## Question 1
 
 Please list the JavaScript primitives and give one example of each.
+Number:	-0
```

Comment:
I'll give you this one, but FYI there is no such thing as -0. By definition, 0 is neither positive nor negative. On the number line, it sits in between -1 and 1.